src/content/docs/data-apis/manage-data/manage-data-retention.mdx

```diff
@@ -342,11 +342,23 @@ This table shows the default retention settings for the available [namespaces](/
   </tbody>
 </table>
 
-## Details for metric and infrastructure data aggregation and retention [#data-details]
+## Metrics and infrastructure data [#data-details]
 
 Here are some details about how metrics and infrastructure data are stored and retained: 
 
 <CollapserGroup>
+
+
+  <Collapser
+    id="dimensional-metrics"
+    title="Dimensional metrics (Metric API)"
+  >
+
+Raw [dimensional metric](/docs/data-apis/understand-data/new-relic-data-types/#dimensional-metrics) data (the `Metric` data type) is aggregated (rolled up) at different time ranges and those aggregations are retained for different lengths of time. Raw sub-minute dimensional metric data is stored for 30 days. 1-minute resolution roll-up data is created and that's retained for 13 months. Only the raw data ingest counts towards usage: the aggregated data does not.
```

Comment:
Will readers know what "sub-minute" dimensional metric data is?

Reply:
good point. lol. it's awkward phrasing.